.github/ISSUE_TEMPLATE/client/catapult/bug.md

```diff
@@ -0,0 +1,41 @@
+---
+name: '[client/catapult] Bug Report'
+about: Got a bug with the client? Fill out this form!
+title: "[client/catapult] Bug Report"
+labels: 'Status: WIP, Type: Bug, catapult'
+assignees: Jaguar0625, gimre-xymcity, Wayonb, 0x6861746366574
+
+---
+
+### System Information
+
+- **Client Version:** 0.0.0
+- **REST Version:** 0.0.0
+- **Operating System:** Windows | MacOS | Linux | Other
+- **Installed With:** Bootstrap | Binaries | From Source
+- **VPS Provider:** AllNodes | Stir Network | Vultr | AWS | Other | None
+- **Are you fully synchronized?** No | Yes
+- **Did you try to restart your node?** No | Yes
+
+#### Description
+
+> A clear and concise description of *what* the bug or issue is. Try to include **actual** versus **expected** behaviour. 
+
+#### Steps to Reproduce
+
+> Please provide detailed steps to reproduce this bug. 
+> [1.] Go to '...'
+> [2.] Click on '...'
+> [3.] Scroll down to '....'
+> [4.] See error. 
+
+#### Logs
+
+> Please attach your logs as text - not as screenshots. You can upload them to [Pastebin](https://pastebin.com/) or as a [Gist](https://gist.github.com/).
```

Comment:
if we want server logs we need to consider few things:
 1. some kind of (optional) log anonymization - I'm not sure if we want that, as things like (node) pubkeys that are present in logs might be valuable for investigation.
  2. there probably will be _multiple_ log files so some might require zip with whole logs dir (and this ofc won't go to pastebin or gist, I guess google drive or anything like that would be most covenient - as long as it's not some shady 3rd party site, think megaupload etc.)
  3. re 2. we'd probably have to archive those _somewhere_ - just for future reference.